examples/conditional_generation/mlflow_peft_adalora_seq2seq.py

```diff
@@ -0,0 +1,194 @@
+import os
+
+import torch
+from datasets import load_dataset
+from torch.utils.data import DataLoader
+from tqdm import tqdm
+from transformers import AutoModelForSeq2SeqLM, AutoTokenizer, default_data_collator, get_linear_schedule_with_warmup
+
+from peft import AdaLoraConfig, PeftConfig, PeftModel, TaskType, get_peft_model
+import mlflow
+
+
+os.environ["TOKENIZERS_PARALLELISM"] = "false"
```

Comment:
Could you explain why this line is necessary? Preferably by a comment linking to the related issue & solution.

Reply:
@loctxmoreh. The original code has them already, I just made a copy of it and have `mlflow` metric lines. For that line of code, I think the author want to make sure there is no `deadlocks` that come from `current process may get forked` (as mentioned in https://stackoverflow.com/questions/62691279/how-to-disable-tokenizers-parallelism-true-false-warning).